src/lib/helpers.js

```diff
@@ -2,6 +2,8 @@ const { TRANCHE_DURATION, BUCKET_DURATION } = require('./constants');
 
 const bnMax = (a, b) => (a.gt(b) ? a : b);
 
+const bnMin = (a, b) => (b.gt(a) ? a : b);
```

Comment:
Was difficult to spot that this is correct as it looked wrong initially. May we switch it with `(a.lt(b) ? a : b)`? Seems more intuitive